lib/src/sip_ua_helper.dart

```diff
@@ -528,10 +531,21 @@ class WebSocketSettings {
   bool allowBadCertificate = false;
 }
 
+class RegisterParams {
+  /// Allow extra headers and Contact Params to be sent on REGISTER
+  /// Mainly used for RFC8599 Support
+  /// https://github.com/cloudwebrtc/dart-sip-ua/issues/89
+  Map<String, dynamic> extraHeaders = {};
```

Comment:
If the `Settings.register_extera_headers` type is `List<String>`, here the `extraHeaders` type should also be `List<String>`

Reply:
Yeah, spotted that after. Reason to be a list?